src/waveform/visualplayposition.cpp

```diff
@@ -88,13 +88,17 @@ double VisualPlayPosition::determinePlayPosInLoopBoundries(
     if (data.m_loopEnabled) {
         double loopSize = data.m_loopEndPos - data.m_loopStartPos;
         if (loopSize > 0) {
-            if ((data.m_playRate < 0.0) && (interpolatedPlayPos < data.m_loopStartPos)) {
+            if ((data.m_playRate < 0.0) &&
+                    (interpolatedPlayPos < data.m_loopStartPos) &&
+                    (data.m_playPos >= data.m_loopStartPos)) {
```

Comment:
Please add a comment, when can this condition be false.
The same below.

Reply:
Done!